kentcdodds/old-kentcdodds.com
# (fix): the layout of workshop and podcast page for extra small devices(iPhone 5/SE)

## PR

### DESCRIPTION

1. The layout of workshop and podcast page was breaking on extra small screens sizes for 
     example iPhone5/SE, it has also been fixed by introducing new media query breakpoint and 
     making some other tweaks.

    New media query breakpoint
    ```
    const maxXS = 320
    export const `@media (max-width: ${maxXS}px`
    ```
2. The warning given below was showing up on the podcast page, it has been fixed.
      **Warning: Each child in a list should have a unique "key" prop.** 

### SCREENSHOT(S) 

#### Workshop broken
![workshop-old](https://user-images.githubusercontent.com/19193724/81483470-06fb4b00-925c-11ea-9050-447a7c6b958c.png)


#### Workshop fixed
![workshop-new](https://user-images.githubusercontent.com/19193724/81483633-45ddd080-925d-11ea-81e4-a1ceb30d76b4.png)


#### Podcast broken
![podcast-old](https://user-images.githubusercontent.com/19193724/81483472-0b276880-925c-11ea-9d0e-a613204e432b.png)


#### Podcast fixed
![podcast-new](https://user-images.githubusercontent.com/19193724/81483641-4ecea200-925d-11ea-9299-ba4712308ab7.png)


### HOW TO TEST

- Go to workshop page, scroll down to the available workshops section, the workshop cards should not expand out of the viewport.
- Go to podcast page the podcast distributor buttons should not expand out of the viewport.
- The podcast play button and heading next to it should be on the same line for extra small display also.
- The unique "key" prop warning should not come on the podcast page console.
